src/components/global-styles.tsx

```diff
@@ -34,6 +34,21 @@ const SizeStyles = css`
         max-width: 392px;
       }
     }
+    .intro-card--item {
+      flex-basis: 50%;
+    }
+
+    .home--action-buttons {
+      flex-direction: row;
+    }
```

Comment:
Do these styles really need to be added globally, rather than described in the respective components?

Reply:
Yes in the component will be better